pkg/api/kptfile/v1/validation_test.go

```diff
@@ -166,23 +166,23 @@ func TestValidateFunctionName(t *testing.T) {
 	}
 	inputs := []input{
 		{
-			"gcr.io/kpt-functions/generate-folders",
+			"gcr.io/kpt-fn/generate-folders",
```

Comment:
Changes in this file doesn't affect the correctness. But I still want to change it to avoid using the old gcr name.